service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersController.java

```diff
@@ -69,6 +71,9 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody Call
                 caseData.getStandardDirectionOrder().getDirections());
 
             directions.forEach((key, value) -> caseDetails.getData().put(key, value));
+
+            caseDetails.getData().put("judgeAndLegalAdvisor",
+                defaultIfNull(caseData.getStandardDirectionOrder().getJudgeAndLegalAdvisor(), null));
```

Comment:
Does this actually change anything? Could you just use `caseData.getStandardDirectionOrder().getJudgeAndLegalAdvisor()`?

Reply:
nope, removed